src/components/Header.jsx

```diff
@@ -4,8 +4,13 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { NavLink } from 'react-router-dom'
 import Config from 'Config'
+import { connect } from 'react-redux'
 
 class Header extends Component {
+  constructor(props) {
+    super(props)
+  }
```

Comment:
this constructor comes along by default, doesn't it?

Reply:
yup.  i put up a small PR to turn on the linter rule that catches this, and cleaned up the 5 newly generated warnings.  https://github.com/LD4P/sinopia_editor/pull/905